Project.toml

```diff
@@ -17,6 +17,13 @@ RegisterUtilities = "d4862ba2-f42c-5aeb-af4f-96a8884a16c4"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[compat]
```

Comment:
Let's add the compat for the packages in the `General` registry too?

Reply:
No, just hard-code the ones that got resolved in your manifest. I.e., 
```
(RegisterFit) pkg> st
Project RegisterFit v0.1.3
Status `~/.julia/dev/RegisterFit/Project.toml`
  [46a7138f] CenterIndexedArrays v0.2.1
⌅ [150eb455] CoordinateTransformations v0.5.1
  [916415d5] Images v0.25.1
  [a98d9a8b] Interpolations v0.13.5
  [2774e3e8] NLsolve v4.5.1
  [429524aa] Optim v1.6.2
  [67712758] RegisterCore v0.2.4
  [464fa2a9] RegisterPenalty v0.3.0
  [d4862ba2] RegisterUtilities v0.1.0
⌅ [90137ffa] StaticArrays v0.12.5
  [37e2e46d] LinearAlgebra
  [10745b16] Statistics
```
and so I would use
```
Images = "0.25"
Interpolations = "0.13"
NLsolve = "4"
Optim = "1"
```
and so on. Notice that I'm not going beyond the first non-zero entry in major/minor/patch.